packages/zk-transfer-web/src/views/MakeTransaction.tsx

```diff
@@ -52,10 +54,15 @@ const MakeTransaction = () => {
       setState({ ...state, sending: false, stage: 1 });
     }, 3000);
   };
+  const demo: any = useContext(Democracy);
+  console.log('The demo before context:', demo);
+  useEffect(() => {
+    console.log('Demo context:', demo);
+  })
 
   return (
     <>
-      <Header />
+      <Header thisAddress={demo.thisAddress || '#'} />
```

Comment:
Good idea to think of a backup value.

Instead of `#` if no address is found, what do you think about changing it to something more obvious like?
```
<Header thisAddress={demo.thisAddress || 'No Ethereum address found.'} />
```
We should never get to the second value normally, unless `await demo.clientInit()` gets interrupted.

Reply:
`No Ethereum address found` is fine. But since the initialization takes sometime, will it be okay if the placeholder remains until the after initalizing.

Reply:
Yes, that's a good suggestion. The code I suggested above will leave this placeholder until initializing is done.